packages/core/admin/admin/src/pages/ProfilePage/index.js

```diff
@@ -180,6 +200,10 @@ const ProfilePage = () => {
     (themeName) => allApplicationThemes[themeName]
   );
 
+  const currentUserRoles = data?.roles?.map(role => role.id) || [];
+  // check if the user role is in the locked roles list
+  const hasLockedRole = dataSSO?.ssoLockedRoles?.some((roleId) => currentUserRoles.includes(Number(roleId))) || false;
```

Comment:
```suggestion
  const hasLockedRole = (dataSSO?.ssoLockedRoles ?? []).some(currentUserRoles.includes);
```

Reply:
ok but it works in this way
`const hasLockedRole = (dataSSO?.ssoLockedRoles ?? []).some((roleId) => currentUserRoles.includes(Number(roleId)));`